### ISSUE TITLE
Nendo server fails immediately on start

### ISSUE THREAD

**Issue by smrl** [okio-ai/nendo-platform#65]
Attached log output.
[error.txt](https://github.com/okio-ai/nendo-platform/files/14552482/error.txt)

Can't connect to running container to investigate further as it immediately shuts down.  Is there something I need to do to enable plugins?  I installed one of the nendo-core plugins as a test but it seems like nendo-core is unrelated to platform -- however it seems I'm now getting CLAP errors as well which was the nendo pip module I installed....?  There is no good documentation explaining how these various repositories interact, if at all.

**Comment by smrl** [comment]
I don't know if it's noteworthy or not but I had a real challenge getting docker set up on this machine (ubuntu 20.04 LTS I believe) -- I started off with the apt repositories, then switched to the Snap version of docker, then installed a recent build of docker-compose from the repository, finally removed it all and installed docker desktop (which runs but doesn't seem to use systemd for the daemon??)...  Just a note, not sure if there may still be funny things with my docker configuration but it looks local to the server itself:

nendo.schema.exception.NendoPluginLoadingError: Failed to import plugin 'nendo_plugin_library_postgres'. Error: [Errno 13] Permission denied: '/home/nendo/nendo_library/ffffffff-1111-2222-3333-1234567890ab'

**Comment by faradox** [comment]
No, you don't need to change anything, following the setup instructions in the Platform's README should be enough to get it running. The CLAP warnings are not an issue, they can be ignored. Since other users have been irritated by them as well, we'll consider changing this behavior in the future.

Now back to your original issue. Assuming you followed the installation steps outlined in the README, you should now see a `library/` directory directly inside the nendo-platform directory. Please report the output of running `ls -l library`. Next, a common mistake is to run any of the `make` commands with `sudo`, you didn't do that by any chance? Please also verify that the use your are running the `make ` commands with is in the `docker` group. You can verify this by running the `groups` command in your shell. You should see `docker` in that list of your user's groups. Please verify all of the aforementioned before we continue digging into this further.

**Comment by smrl** [comment]
I've verified that I'm using make as a user that's a member of the docker group.  My library directory is empty.  I tried removing my entire docker environment to see if I could install the most recent docker using their documentation, so I installed most recent docker-engine instead.  This would not build and complained "services.web-http: container name nendo-web is already in use by services.web: invalid compose project"

This was after purging and verifying that there was nothing remaining from a prior docker installation.  The only way I've successfully built this is with docker-desktop.  After trying some more at making sure my docker environment was correct, I purged everything again and reinstalled docker-desktop, downloaded a fresh copy of the repo and built again and reproduced the same error.  I did not note any errors appear during the build process.

**Comment by faradox** [comment]
> My library directory is empty.

Please verify that it's also owned by the user with which you are running docker. Do `ls -l | grep library` in the platform's repository root to verify that your user owns the directory.

> "services.web-http: container name nendo-web is already in use by services.web: invalid compose project"

To fix that, try `make stop ; docker container prune ; docker builder prune`. Then try to build again.

I'm not really sure about docker-desktop, all our tests have been performed with docker-engine.
